lib/records/fhir_dstu2.rb

```diff
@@ -332,6 +353,29 @@ def self.observation(observation, fhir_record, patient, encounter)
         fhir_record.entry << entry
       end
 
+      def self.quality_of_life_observation(observation, fhir_record, patient)
```

Comment:
I realize you are following the same pattern between the FhirRecord and FhirDstu2Record, but the code here is mostly the same. It would be great if you could refactor this code so that it reused most of what is in FhirRecord. That pattern could then be repeated across the rest of the code in this file.